### PR TITLE
Require Maven 3.2.5+

### PR DESCRIPTION
**Pull request by slachiewicz**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -537,8 +537,8 @@ limitations under the License.
             <configuration>
               <rules>
                 <requireMavenVersion>
-                  <version>3.0.5</version>
-                  <message>This project requires at least Maven 3.0.5</message>
+                  <version>3.2.5</version>
+                  <message>This project requires at least Maven 3.2.5</message>
```
Comment (slawekjaranowski) [inline pom.xml:541]: It is requirement for build projects, so I would like see something newer, eg 3.6.x, 3.8.x

Comment (slachiewicz) [reply pom.xml:541]: In theory yes, but that will force us to drop 3.2.5 from ci test matrix

Comment (slawekjaranowski) [reply pom.xml:541]: We should agree on build matrix - compared to maintained Maven versions ... and so on ... but it is another discussion

**Review by slawekjaranowski — APPROVED** [review]

**Comment by olamy** [comment]
not sure to understand why do we have 3.2.5+ here but 3.5.4+ for compiler https://github.com/codehaus-plexus/plexus-compiler/pull/283#pullrequestreview-1407002186 ?
any reason for this and not having parent 3.5.4+ so components will be upgraded when upgrading parent component?

**Comment by slachiewicz** [comment]
Becouse when I started with upgrade, many issues where rised. See logs from GHA

**Comment by olamy** [comment]
I would say let's upgrade parent directly to 3.5.4+ same as for compiler.
or we will need to upgrade again in few months and maybe do the job twice.
which logs from GHA? I can't see anything in this PR .

**Pull request merged**